lib/Sema/TypeCheckNameLookup.cpp

```diff
@@ -555,13 +555,21 @@ void TypeChecker::performTypoCorrection(DeclContext *DC, DeclRefKind refKind,
 }
 
 static InFlightDiagnostic
-diagnoseTypoCorrection(TypeChecker &tc, DeclNameLoc loc, ValueDecl *decl) {
+diagnoseTypoCorrection(TypeChecker &tc,
+                       DeclNameLoc loc, 
+                       ValueDecl *decl, 
+                       FunctionRefKind refKind) {
+  
+  auto name = refKind == FunctionRefKind::Compound ? decl->getFullName() :
```

Comment:
LLVM style moves the colon to a new line in-line with the else condition and aligns it with the question mark.